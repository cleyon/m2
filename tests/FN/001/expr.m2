Addition: 2+2=@expr 2+2@
Math: Sqrt(2)=@expr sqrt(2)@
@define X 13
15 times 13 is @expr 15*X@
