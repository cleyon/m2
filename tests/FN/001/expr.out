Addition: 2+2=4
Math: Sqrt(2)=1.41421
15 times 13 is 195
